build/Makefile.kits

```diff
@@ -213,11 +229,17 @@ else # ifneq ($(COMBINED_PACKAGES),1)
 	# (Copying for combined packages)
         ifeq ($(DISABLE_LISTENER),0)
```

Comment:
It seems DISABLE_LISTENER is passed to create_bundle as value for provider only build. We are planning to remove provider only option. I will remove all references to DISABLE_LISTENER in that commit.